src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -96,21 +110,17 @@ public void onClassificationChange(Location l, NetworkComponent from, NetworkCom
 
         if (component != null) {
             switch (component.getEnergyComponentType()) {
-                case CAPACITOR:
```

Comment:
It’s probably fine to keep this in, but in the future try to keep the scope of you changes limited to the change itself. Other things like this should be in a separate PR. don’t change this now though unless someone else has an issue with this.

Reply:
No need to remove, it's a good change. I was stating it for the future. Now it's done may as well keep it in :)